products/artifactory.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Artifactory
+category: server-app
+permalink: /Artifactory
+alternate_urls:
+-   /artifactory
+changelogTemplate: https://jfrog.com/help/r/jfrog-release-information/artifactory-__RELEASE_CYCLE__
+iconSlug: jfrog
+releases:
+-   releaseCycle: "7.59"
+    eol: 2024-11-28
```

Comment:
`releaseDate`s are missing for all version. That information is available for 7.x on https://jfrog.com/help/r/jfrog-release-information/artifactory-7.0.1-cloud-self-hosted.